schemas/app-sre/integration-1.yml

```diff
@@ -37,37 +56,65 @@ properties:
 
   pr_check:
     type: object
+    description: |
+      Configuration for pull request checks related to this integration.
+      This section defines how PR checks are executed, their state, and
+      additional options for validation and execution order.
     additionalProperties: false
     properties:
       cmd:
         type: string
+        description: |
+          The command to execute for the PR check.
       state:
         type: boolean
+        description: |
+          Indicates if the PR check is stateful.
       sqs:
         type: boolean
+        description: |
+          Whether to send PR check results to SQS.
```

Comment:
Maybe to keep consistency with the other descriptions, we can reword these descriptions to start with `If true,`

Reply:
```suggestion
      state:
        type: boolean
        description: |
          If true, PR check job is stateful.
      sqs:
        type: boolean
        description: |
          If true, send PR check results to SQS.
```